packages/cxl-lumo-styles/scss/_mixins.scss

```diff
@@ -43,3 +43,49 @@
     max-width: $max-width;
   }
 }
+
+@mixin wide-background {
+  position: relative;
+  overflow-x: hidden;
+  background-repeat: no-repeat !important;
+  background-size: auto clamp(65%, 40vw, 90%) !important;
+
+  &::before {
+    position: absolute;
+    bottom: 0;
+    z-index: -1;
+    width: 100%;
+    height: 100%;
+    content: "";
+    border-radius: calc(6 * var(--lumo-border-radius-l))
+
+  }
+}
+
+@mixin extend-background-left {
+  background-position: bottom right calc(50vw + clamp(48px,5%,100px)) !important;
+
+  &::before {
+    right: 47%;
+    height: 60%;
```

Comment:
Yes, @heshfekry and I tested them during a call, to try matching them to the design. However, those values have since indeed changed, to improve visual balance and make the hero image more prominent.